src/libs/actions/BankAccounts.js

```diff
@@ -270,12 +272,352 @@ function fetchUserWallet() {
         });
 }
 
+let previousACHData = {};
+Onyx.connect({
+    key: ONYXKEYS.REIMBURSEMENT_ACCOUNT,
+    callback: (val) => {
+        previousACHData = lodashGet(val, 'achData', {});
+    },
+});
+
+function goToWithdrawalStepID(stepID, achData) {
```

Comment:
Some of these decisions are about trying to keep things familiar since a handful of people have worked on this code historically. The equivalent method in Web-Secure is [`goToStepID()`](https://github.com/Expensify/Web-Secure/blob/044c82affb78812a58b881a6d5ba026d91dace3b/site/app/dialogs/reimbursementAccount/reimbursementAccountPage.jsx#L160-L165). So I tried to keep it somewhat familiar when converting to an action (but needed to give it some kind of context in the `BankAccounts.js` file so that it is somewhat more specific.

Reply:
That said, maybe this is a good time to scrap it and give it a name that reflects what it does which is update the account data locally which will navigate to a new step as a side effect.